source/2020-02-28-the-ember-times-issue-137.md

```diff
@@ -25,13 +27,15 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Ember Website Redesign 🔥](https://emberjs.com/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Maybe you haven't noticed, but the **primary website for Ember** has been **redesigned** and it is **now live**! The redesign brings a fresh new look at the main piece of real estate for Ember on the interent and features a clean and modern look that all of us in the community can be proud of. 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+This redesign has been a long time in the making and has invovled a lot of work from the people across the Ember teams and the Ember community. Big shout outs to everyone involved in the effort, especially [Mel Sumner @melsumner](https://github.com/MelSumner) and [Chris Manson @mansona](https://github.com/mansona)! 🎉🎉🎉 It's super exciting to see this modern re-visioning of the website come to life.
```

Comment:
```suggestion
This redesign has been a long time in the making and has invovled a lot of work from the people across the Ember teams and the Ember community. Big shout outs to everyone involved in the effort, especially [Mel Sumner @melsumner](https://github.com/MelSumner) and [Chris Manson @mansona](https://github.com/mansona)! 🎉🎉🎉 It's super exciting to see this modern vision of the website come to life.
```
For me re-visioning feels like a really complex word which I cannot completely place, so vision might be easier